Ghost_in_the_Shell/client_folder_react_app/ethisim/.eslintrc.json

```diff
@@ -29,5 +29,6 @@
      * "error" or 2 - turn the rule on as an error (exit code will be 1)
      * There could be other rules not added below that should be added
      */
+     "react/display-name": "off"
```

Comment:
what does this do?

Reply:
I had eslint errors for this, I don't think it's important